### ISSUE TITLE
Modular crusher coal ore incorrect

### ISSUE THREAD

**Issue by Githubaccountlol** [Beyond-Reality/Beyond-Reality-Divergence#233]
NEI lists 4 coal as the result of crushing coal ore, but you get dusts last time I checked.  IIRC only 1 dust as well.

**Comment by Dominance** [comment]
Yeah this is a mod issue IIRC will look into it. I seen something similar when adding Coal Dust to the railcraft crusher and trying to fix Modular systems.
